allure-report-face/src/components/widgets-grid/WidgetsGridView.js

```diff
@@ -7,7 +7,9 @@ import {className} from '../../decorators';
 import allurePlugins from '../../pluginApi';
 
 const widgetTpl = (id) => `<div class="widget island" data-id="${id}">
-    <div class="widget__handle fa fa-arrows"></div>
+    <div class="widget__handle">
+        <span class="fa fa-ellipsis-v"></span> <span class="fa fa-ellipsis-v"></span>
```

Comment:
No one of my automate engeneers knows about ability to move widgets. But all of them know how to move steps in jenkins. Jenkins uses vertical six points (`:::`) to indicate movable state.